Makefile

```diff
@@ -114,12 +114,12 @@ test-json-format-output:
 # Run component e2e tests
 .PHONY: test-cmp-e2e
 test-cmp-e2e:
-	go test -v github.com/openshift/odo/tests/integration --ginkgo.focus="odoCmpE2e" -ginkgo.slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -ginkgo.v -timeout $(TIMEOUT)
+	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odoCmpE2e" slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
```

Comment:
why are we now randomizing? you need to document this in your PR description / commit.

Reply:
https://github.com/openshift/odo/pull/1797/files#diff-b67911656ef5d18c4ae36cb6741b7965R114